### PR TITLE
[devbin] Automate cloning db and accessing it with a preconfigured VM

### PR DESCRIPTION
**Pull request by jigold**
This PR creates a clone of a database along with a companion VM where the SSL and MySQL user credentials are setup automatically.

Note that you'll have to run the script multiple times to get the final result as the "clone" step will time out. The script should be idempotent.

### CODE REVIEW

**Comment by danking** [comment]
I dismissed them as won’t fix because they’re leaking secrets as intended into config files.

**Comment by danking** [comment]
@daniel-goldstein bump

File: devbin/setup-vm-with-clone-db-access.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+mkdir /sql-config/
+chmod 777 /sql-config/
+
+git clone https://github.com/hail-is/hail.git
+cd hail
+
+pip3 install -r hail/python/requirements.txt
+pip3 install -r hail/python/dev/requirements.txt
+pip3 install -r hail/python/hailtop/requirements.txt
+pip3 install -r gear/requirements.txt
```
Comment (daniel-goldstein) [inline devbin/setup-vm-with-clone-db-access.sh:17]: We try to use `pinned-requirements.txt` files as much as possible for reproducibility. You can instead do

```
pip install \
    -r hail/python/pinned-requirements.txt \
    -r hail/python/dev/pinned-requirements.txt \
    -r gear/pinned-requirements.txt
```

**Review by daniel-goldstein — CHANGES_REQUESTED** [review]

**Comment by jigold** [comment]
I'm going to close this for now and address your comments the next time I need to clone the database.

**Pull request closed without merging**